spec/openapi.yaml

```
@@ -276,6 +276,33 @@ paths:
             text/xml:
               schema:
                 type: string
+  "/v{version}/deleteImage/{image_id}":
+    post:
+      tags:
+        - Articles
+      summary: Delete image for article
+      description: This will remove an image
+      operationId: deleteImage
+      parameters:
+        - $ref: "#/components/parameters/version"
+        - $ref: "#/components/parameters/language"
+        - $ref: "#/components/parameters/format"
+        - name: image_id
+          in: path
+          description: Image id
+          required: true
+          schema:
+            type: integer
+      responses:
+        "204":
+          description: success
+          content:
+            application/json:
+              schema:
+                type: string
+            text/xml:
+              schema:
+                type: string
```

```suggestion
        "204":
          description: Success, empty response
```

The 204 response does not have content so the docs should not say that is has ;)